ilc/client/BundleLoader.js

```diff
@@ -0,0 +1,82 @@
+import crashIlc from './errorHandler/crashIlc';
+
+export class BundleLoader {
+
+    #cache = new WeakMap();
```

Comment:
Please add WeakMap into the URL that we use to build polyfills with the help of `build:polyfills` script

Reply:
I they're supported even in IE 11. Are you sure we need polyfill for them?

Reply:
Discussed